packages/apollo-link-schema/src/schemaLink.ts

```diff
@@ -40,7 +33,7 @@ export class SchemaLink extends ApolloLink {
           this.schema,
           operation.query,
           this.rootValue,
-          this.context,
+          operation.getContext(),
```

Comment:
Sorry to bump this, @peggyrayzis I think this is gtg and It's really holding me back :( Thank you!